GUI/MainWindow.xaml.cs

```diff
@@ -500,17 +500,24 @@ private void EverythingRunnerExceptionHandler(Task obj)
                 var messageBoxResult = System.Windows.MessageBox.Show(message + "\n\nWould you like to report this crash?", "Runtime Error", MessageBoxButton.YesNo);
                 outRichTextBox.AppendText(message + Environment.NewLine);
                 Exception exception = e;
-
+                //Find Output Folder
+                string outputFolder = e.Data["folder"].ToString();
+                var toml = Directory.GetFiles(outputFolder, "*.toml");
```

Comment:
check to make sure the directory exists here before continuing, by using Directory.Exists